Friendly-plans/app/build.gradle

```diff
@@ -43,6 +43,7 @@ dependencies {
     compile "com.google.dagger:dagger:2.9"
     annotationProcessor "com.google.dagger:dagger-compiler:2.9"
     provided 'javax.annotation:jsr250-api:1.0'
+    compile 'com.android.support:recyclerview-v7:24.2.0'
```

Comment:
I think some fresh commits from master are not merged in your branch. There are some important changes (e.g. greendao schema section is missing), please try merge them. If you would have any problems let me know :)